libraries/libfc/include/fc/io/random_access_file.hpp

```diff
@@ -394,7 +407,7 @@ class random_access_file {
       std::streamsize read(char* s, std::streamsize n) {
          ssize_t total_red = 0;
          while(n - total_red) {
-            ssize_t red = ctx->read_from(boost::asio::buffer(s+total_red, n-total_red), pos);
+            ssize_t red = ctx->read_from(mutable_buffers_1(s+total_red, n-total_red), pos);
```

Comment:
No need to change this line, the problem can be solved above in 1 line,
```diff
--- a/libraries/libfc/include/fc/io/random_access_file.hpp
+++ b/libraries/libfc/include/fc/io/random_access_file.hpp
@@ -95,7 +95,7 @@ struct random_access_file_context {
    ssize_t read_from(const MutableBufferSequence& mbs, ssize_t offs) {
       struct iovec iov[IOV_MAX];
       int i = 0;
-      for(const boost::asio::mutable_buffer& b : mbs) {
+      for(const auto& b : boost::beast::buffers_range(mbs)) {
          iov[i].iov_base = b.data();
          iov[i].iov_len = b.size();
          if(++i == IOV_MAX)
```

Reply:
Thanks! Even better!

Reply:
https://github.com/AntelopeIO/spring/pull/1384/commits/f12b0a19ebd92589e30447dfddd9c53fb94c0f20